intelmq/bots/outputs/restapi/output.py

```diff
@@ -34,7 +34,13 @@ def process(self):
                               cert=self.ssl_client_cert,
                               timeout=self.http_timeout_sec,
                               **kwargs)
-        r.raise_for_status()
+        if self.parameters.logging_level in ["DEBUG", 10] and r.status_code != 200:
```

Comment:
Logging level should be never 10, if it's not that's a bug.
And when using `logger.debug` the message is only logged if the level is debug or lower, so we don't need to check that.
And to check the response, better use `r.ok` (returns False if status code is between 400 and 600).

Reply:
I tried to access the variable logging_level and when not set directly for the bot it's content is the integer representation of given level (10 in this case). I will get rid of the level check and keep only the logging statement and r.ok check.

Reply:
> I tried to access the variable logging_level and when not set directly for the bot it's content is the integer representation of given level (10 in this case).

That's interesting, I will check that.